worker/transcribee_worker/whisper_transcribe.py

```diff
@@ -0,0 +1,179 @@
+import asyncio
+import logging
+from typing import Iterator, Tuple
+
+import requests
+from numpy.typing import ArrayLike
+from transcribee_proto.document import UNKNOWN_SPEAKER, Atom, Paragraph
+from transcribee_worker.config import MODELS_DIR
+from whispercpp import api
+
+
+def get_model_file(model_name: str):
+    whisper_models_dir = MODELS_DIR / "whisper"
+    whisper_models_dir.mkdir(parents=True, exist_ok=True)
+    model_file = whisper_models_dir / f"{model_name}.bin"
+
+    if not model_file.exists():
+        logging.info(f"downloading model {model_name} because it does not exist yet...")
+        base_url = "https://huggingface.co/datasets/ggerganov/whisper.cpp/resolve/main"
+        url = f"{base_url}/ggml-{model_name}.bin"
+        r = requests.get(url, allow_redirects=True)
+        r.raise_for_status()
+        with model_file.open(mode="wb") as f:
+            f.write(r.content)
+
+    return model_file
+
+
+def get_context(model_name: str) -> api.Context:
+    model_file = get_model_file(model_name)
+    logging.info(f"loading model {model_name}...")
+    ctx = api.Context.from_file(str(model_file))
+    ctx.reset_timings()
+    return ctx
+
+
+class TranscriptionWorkDoneToken:
+    pass
+
+
+# TODO(robin): this currently filters all special tokens
```

Comment:
can you remove your name there?

Reply:
& open an issue once this is merged?

Reply:
Why remove the name?
That way you save one `git blame` to figure out who to ask about a `TODO`.

Reply:
seems just not super nescessary to me here but no strong feelings on that